<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelpSettingsDialog</class>
 <widget class="QDialog" name="HelpSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Help for settings</string>
  </property>
  <widget class="QPushButton" name="pushButtonLast">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>320</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNext">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>320</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNext">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>290</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLast">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>290</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Last</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfo1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>431</width>
     <height>321</height>
    </rect>
   </property>
   <property name="text">
    <string>Speed freak communicates with server.
If you want send your results to server
or view other users results you need to
register to server.

1. For registering: Set user name,
password and valid e-mail address to
the fields. Press Register-button and
your registering will be confirmed.   </string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSettingsPic">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>50</y>
     <width>291</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="graphics.qrc">:/new/prefix1/Graphics/settingsDialog.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfo2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>421</width>
     <height>351</height>
    </rect>
   </property>
   <property name="text">
    <string>2. After succesful registering you need
to put username and password to the
fields and press Log in -button. After
that you are logged in and can send
own results to the server.

NOTE: If you are registered your user-
name before you just need to set your
username and password in to the Set user
fields and press Log in -button.</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="graphics.qrc"/>
 </resources>
 <connections/>
</ui>
